src/index.js

```diff
@@ -316,10 +317,22 @@ aws.config.region = "${region}";
 const lambda = new aws.Lambda();
 const functions = ${JSON.stringify(functions)};
 
+Promise.delay = function(t, val) {
+  return new Promise(resolve => {
+    setTimeout(resolve.bind(null, val), t);
+  });
+};
+
+Promise.raceAll = function(timeoutTime, timeoutVal, promises) {
+  return Promise.all(promises.map(p => {
```

Comment:
This is the exact same thing as

```js
  return Promise.race([ Promise.delay(timeoutTime, timeoutVal), ...promises ]);
```

but creating a lot more intermediate promises.

Reply:
Good catch. Again, my blind C&P from SO was a bad move.  I'll get that cleaned up when I revise this PR.